docs/DDL-Operations-on-Carbon.md

```diff
@@ -67,6 +67,14 @@ Here, DICTIONARY_EXCLUDE will exclude dictionary creation. This is applicable fo
   ```ruby
   TBLPROPERTIES ("COLUMN_GROUPS"="(column1,column3),(Column4,Column5,Column6)") 
   ```
+ - **Table Block Size Configuration**
+
+   The block size of one table's files on hdfs can be defined using an int value whose size is in MB, the range is form 1MB to 2048MB and the default value is 1024MB, if user didn't define this values in ddl, it would use default value to set.
+
+  ```ruby
+  TBLPROPERTIES ("TABLE_BLOCKSIZE"="512")
```

Comment:
I think it is better to have the MB inside the value, it is more clear to the user. For example:
`TBLPROPERTIES ("TABLE_BLOCKSIZE"="512MB")`
And we can also add support for GB in the future, if required. 
Can you modify the code accordingly